src/components/Offers/New/CreateOfferForm.spec.js

```diff
@@ -204,13 +203,37 @@ describe("Create Offer Form", () => {
 
             await fireEvent.click(screen.getByText("Advanced Settings"));
 
-            expect(screen.queryByText("Publication Date")).toBeVisible();
-            expect(screen.queryByText("Publication End Date")).toBeVisible();
+            expect(screen.queryByText("Publication Date *")).toBeVisible();
+            expect(screen.queryByText("Publication End Date *")).toBeVisible();
             expect(screen.queryByText("Hide offer")).toBeVisible();
         });
 
-    });
+        it("should not have job type options with null value", async () => {
+            useSession.mockImplementation(() => ({ isLoggedIn: true, data: { company: { name: "Company Name" } } }));
 
+            renderWithStoreAndTheme(
+                <BrowserRouter>
+                    <MuiPickersUtilsProvider utils={DateFnsUtils}>
+                        <CreateOfferWrapper>
+                            <CreateOfferPage />
+                        </CreateOfferWrapper>
+                    </MuiPickersUtilsProvider>
+                </BrowserRouter>,
+                { initialState, theme }
+            );
+
+            await act(async () => {
+                const button = await screen.findByLabelText("Job Type *");
+                fireEvent.mouseDown(button);
+            });
+
+            const elements = await screen.findAllByTestId("job-type-item");
+            elements.forEach((element) => {
+                expect(element).toHaveAttribute("data-value");
+                expect(element).toBeVisible();
+            });
```

Comment:
Instead of test id, maybe something like `JobOptions.forEach((key, val) => screen.findByText(val)` would work

Reply:
Please check out [this comment](https://github.com/NIAEFEUP/nijobs-fe/pull/159#discussion_r825454975)